mailbox/plugin/quota-search-elasticsearch/src/main/java/org/apache/james/quota/search/elasticsearch/json/QuotaRatioToElasticSearchJson.java

```diff
@@ -42,9 +43,7 @@ public String convertToJson(String user, QuotaUsageUpdatedEvent event) throws Js
         return mapper.writeValueAsString(QuotaRatioAsJson.builder()
                 .user(user)
                 .domain(event.getQuotaRoot().getDomain().map(Domain::asString))
-                .quotaRatio(Math.max(
-                    event.getCountQuota().getRatio(),
-                    event.getSizeQuota().getRatio()))
+                .quotaRatio(QuotaRatio.from(event.getSizeQuota(), event.getCountQuota()).max())
```

Comment:
I think there's something wrong here, you have a quotaRatio builder method, you build a QuotaRatio object, but you don't pass it to the method.
Maybe that's because QuotaRatio should be a (typed) double ?